how-to-guides/how-it-works/scripts/Neptune_API_Tour_querying.py

```diff
@@ -1,18 +1,22 @@
 import neptune.new as neptune
 
 # download runs table from Neptune
-my_project = neptune.get_project(name="common/quickstarts", api_token="ANONYMOUS")
+my_project = neptune.init_project(name="common/quickstarts", api_token=neptune.ANONYMOUS_API_TOKEN)
```

Comment:
If this script only does querying, we can use `mode="read-only"` in each init call.